Source/Scene/ModelInstanceCollection.js

```diff
@@ -59,7 +59,7 @@ import ShadowMode from './ShadowMode.js';
      * @param {Boolean} [options.incrementallyLoadTextures=true] Determine if textures may continue to stream in after the model is loaded.
      * @param {ShadowMode} [options.shadows=ShadowMode.ENABLED] Determines whether the collection casts or receives shadows from each light source.
      * @param {Cartesian2} [options.imageBasedLightingFactor=new Cartesian2(1.0, 1.0)] Scales the diffuse and specular image-based lighting from the earth, sky, atmosphere and star skybox.
-     * @param {Cartesian3} [options.lightColor] The color and intensity of the sunlight used to shade models.
+     * @param {Cartesian3} [options.lightColor] The light color and intensity when shading models. When <code>undefined</code> the scene's light color and intensity are used instead.
```

Comment:
Consider calling it `light color` instead of `light color and intensity` to reduce confusion. This pops up in other places too.